x/sportevent/keeper/msg_server_add_event.go

```diff
@@ -83,17 +83,10 @@ func initBetConstraints(event *types.SportEvent, params types.Params) bool {
 	// init all Bet constraints if nil
 	if event.BetConstraints == nil {
 		event.BetConstraints = &types.EventBetConstraints{
-			MaxBetCap:      params.EventMaxBetCap,
```

Comment:
max bet cap, min amount, betfee constraints should not be removed
??

Reply:
removed